s3_tar_ball.go

```diff
@@ -1,22 +1,20 @@
 package walg
 
 import (
-	"io"
 	"archive/tar"
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 	"github.com/pkg/errors"
+	"io"
 	"log"
-	"github.com/pierrec/lz4"
-	"encoding/json"
-	"github.com/aws/aws-sdk-go/service/s3/s3manager"
-	"github.com/aws/aws-sdk-go/aws"
-	"bytes"
 )
 
 // S3TarBall represents a tar file that is
 // going to be uploaded to S3.
 type S3TarBall struct {
-	baseDir          string
 	trim             string
 	bkupName         string
```

Comment:
I'm surprised this didn't trigger you :)

Reply:
Fixed :)